extensions/cornerstone/src/services/SegmentationService/SegmentationService.ts

```diff
@@ -416,6 +417,14 @@ class SegmentationService extends PubSubService {
 
     segDisplaySet.images = derivedSegmentationImages;
 
+    segDisplaySet.images = segDisplaySet.images.map(image => {
```

Comment:
let me know if this is ok.

Reply:
Are you proposing deleting the previous line then?

Reply:
yes i'll remove